stdr_parser/src/stdr_yaml_parser.cpp

```diff
@@ -133,19 +135,19 @@ namespace stdr_parser
             parser.GetNextDocument(doc);
           #endif
           new_node->file_origin = file_name;
-#ifndef HAVE_NEW_YAMLCPP
-          new_node->file_row = doc.GetMark().line;
-#endif
+          #ifndef HAVE_NEW_YAMLCPP
+            new_node->file_row = doc.GetMark().line;
+          #endif
           
           parseLow(doc,new_node);
         }
         else
         {
-#ifdef HAVE_NEW_YAMLCPP
+        #ifdef HAVE_NEW_YAMLCPP
           parseLow(it->second,new_node);
-#else
+        #else
           parseLow(it.second(),new_node);
-#endif
+        #endif
```

Comment:
The indentation was that way, so when the code passes through the pre-processor and the `#ifndef/#endif` statements will be removed, the remaining code will have the correct indentation.